src/cpp/meschach/mesch12a/src/machine.c

```diff
@@ -99,39 +99,39 @@ register int	len;
 
 /* __smlt__ scalar multiply array c.f. sv_mlt() */
 void	__smlt__(dp,s,out,len)
-register Real	*dp, *out;
-register double s;
-register int	len;
+ Real	*dp, *out;
+ double s;
+ int	len;
```

Comment:
prototype : add the parameters in the function def

Reply:
done